### PR TITLE
terraform/cloudflare: add dns for darwin builder

### PR DESCRIPTION
**Pull request by zowoq**

### CODE REVIEW

**Review by zimbatm — APPROVED** [review]

File: terraform/cloudflare_nix-community_org.tf [hunk 1]
```diff
@@ -58,6 +58,20 @@ resource "cloudflare_record" "nix-community-org-build04-AAAA" {
   type    = "AAAA"
 }
 
+resource "cloudflare_record" "nix-community-org-darwin01-A" {
+  zone_id = local.nix_community_org_zone_id
+  name    = "darwin01"
+  value   = "?"
```
Comment (0x4A6F) [inline terraform/cloudflare_nix-community_org.tf:64]:
```diff
-  value   = "?"
+  value   = "167.235.14.165"
```

File: terraform/cloudflare_nix-community_org.tf [hunk 2]
```diff
@@ -58,6 +58,20 @@ resource "cloudflare_record" "nix-community-org-build04-AAAA" {
...
+  type    = "A"
+}
+
+resource "cloudflare_record" "nix-community-org-darwin01-AAAA" {
+  zone_id = local.nix_community_org_zone_id
+  name    = "darwin01"
+  value   = "?"
```
Comment (0x4A6F) [inline terraform/cloudflare_nix-community_org.tf:71]: Currently there is no AAAA record.

**Review by 0x4A6F — CHANGES_REQUESTED** [review]

File: terraform/cloudflare_nix-community_org.tf (hunk 2 above)
Comment (0x4A6F) [reply terraform/cloudflare_nix-community_org.tf:71]:
```diff
-  value   = "?"
+  value   = "2a01:4f8:262:1d98::1"
```

Comment (zowoq) [reply terraform/cloudflare_nix-community_org.tf]: Thanks.

**Review by Mic92 — APPROVED** [review]

**Pull request merged**